revue-francaise-d-administration-publique.csl

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Revue française d'administration publique (French)</title>
+    <title-short>RFAP</title-short>
+    <id>http://www.zotero.org/styles/revue-francaise-d-administration-publique</id>
+    <link href="http://www.zotero.org/styles/revue-francaise-d-administration-publique" rel="self"/>
+    <link href="http://www.zotero.org/styles/revue-francaise-de-gestion" rel="template"/>
+    <link href="https://www.ena.fr/La-recherche-a-l-ENA/Publications/rfap" rel="documentation"/>
+    <author>
+      <name>Romain Pierronnet</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <issn>0152-7401</issn>
+    <eissn>1965-0620</eissn>
+    <summary>Le style utilisé en 2019 par la Revue française d'administration publique.</summary>
+    <updated>2018-10-04T23:58:26+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">
+        <single>dir.</single>
+        <multiple>dirs.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=" ">
+      <name name-as-sort-order="all" sort-separator=" " delimiter=", " delimiter-precedes-last="always" initialize-with="." font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
+      <substitute>
+        <names variable="editorial-director"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-last">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " delimiter=" ; " delimiter-precedes-last="always" font-variant="normal" font-weight="normal"/>
+      <label form="short" prefix=" " text-case="lowercase"/>
+      <substitute>
+        <names variable="editor">
+          <name name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="always" initialize-with="." font-variant="small-caps"/>
```

Comment:
is this on purpose? Editors and authors are formatted differently (e.g. small caps vs. not, semicolong vs. comma, etc.)? That seems very odd.